api/app/src/main/kotlin/packit/controllers/RoleController.kt

```diff
@@ -0,0 +1,25 @@
+package packit.controllers
+
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.stereotype.Controller
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import packit.model.dto.CreateRole
+import packit.service.RoleService
+
+@Controller
+@PreAuthorize("hasAuthority('user.manage')")
+@RequestMapping("/role")
+class RoleController(private val roleService: RoleService)
+{
+    @PostMapping()
+    fun createRole(@RequestBody @Validated createRole: CreateRole): ResponseEntity<Map<String, String?>>
+    {
+        roleService.createRole(createRole)
+
+        return ResponseEntity.ok(mapOf("message" to "Role created"))
```

Comment:
yup agree.. have created dtos needed for this in user endpoints so for now have created ticket https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-5324/Create-endpoints-to-return-Dtos... this will get done after endpoints tickets